build/scripts/meta.yaml.schema

```diff
@@ -33,7 +33,7 @@
     },
     "icon": {
       "type": "string",
-      "pattern": "[-._a-zA-Z0-9]+?\\.(?:svg|png)(?:\\?.*)?$"
+      "pattern": "[-_a-zA-Z0-9]+?(?:\\.(?:svg|png))?(?:\\?.*)?$"
```

Comment:
I know regexes are generally write-only, but I'm having trouble understanding the change here. If `.svg` or `.png` is optional, it should be removed -- this regex reduces to `[-_a-zA-Z0-9]$`.

Reply:
You are right, we can omit checking for the extension here.